.github/workflows/ci_main.yml

```diff
@@ -126,15 +127,19 @@ jobs:
         with:
           filters: |
             simpledbus: &simpledbus
-              - '.github/workflows/**'
-              - '.github/actions/**'
               - 'simpledbus/**'
             simplebluez: &simplebluez
               - *simpledbus
               - 'simplebluez/**'
             simpleble: &simpleble
               - *simplebluez
               - 'simpleble/**'
+      - name: Debug Output
```

Comment:
I'll merge this PR, but can we remove this in a follow-up PR?

Reply:
Agree, please check https://github.com/OpenBluetoothToolbox/SimpleBLE/pull/358